### PR TITLE
Wire up Datadog Tracing

### PR DESCRIPTION
**Pull request by devinmatte**
## Motivation

#380 

Give us more detailed information about the performance of the app

## Changes

- Adds datadog lambda tracing to chalice

### CODE REVIEW

File: server/frontend-cfn.json [hunk 1]
```diff
@@ -80,7 +92,10 @@
       "Properties": {
         "Environment": {
           "Variables": {
-            "MBTA_V2_API_KEY": { "Ref": "MbtaV2ApiKey" }
+            "MBTA_V2_API_KEY": { "Ref": "MbtaV2ApiKey" },
+            "DD_SITE": { "Ref": "DDSite" },
```
Comment (mathcolo) [inline server/frontend-cfn.json:96]: How likely is it that DD_SITE will change? perhaps we can just hardcode that?

File: deploy.sh [hunk 2]
```diff
@@ -90,7 +90,8 @@ aws cloudformation deploy --template-file cfn/packaged.yaml --stack-name $CF_STA
     TMFrontendHostname=$FRONTEND_HOSTNAME \
     TMFrontendCertArn=$FRONTEND_CERT_ARN \
     TMBackendCertArn=$BACKEND_CERT_ARN \
-    MbtaV2ApiKey=$MBTA_V2_API_KEY
+    MbtaV2ApiKey=$MBTA_V2_API_KEY \
+    DDApiKey=$DD_API_KEY
```
Comment (mathcolo) [inline deploy.sh:94]: Missing DD_SITE here

File: server/frontend-cfn.json (hunk 1 above)
Comment (mathcolo) [reply server/frontend-cfn.json:96]: (and if you choose to do so, vars that are static can be hard-coded here https://github.com/transitmatters/t-performance-dash/blob/dashboard-v4/server/.chalice/config.json#L13)

**Review by mathcolo — COMMENTED** [review]
exceedingly close

File: server/frontend-cfn.json (hunk 1 above)
Comment (devinmatte) [reply server/frontend-cfn.json:96]: Likely to never change, I'll do that then

**Review by mathcolo — APPROVED** [review]

**Pull request merged**